app/src/main/java/com/example/fivenightsescape/MapsActivity.kt

```diff
@@ -28,13 +32,33 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     private lateinit var fusedLocationClient: FusedLocationProviderClient
     private lateinit var mMap: GoogleMap
 
+    //timer and lv things
+    private var levelChanger = 4 //levelChanger * delay = every how many milliseconds next lv, it changes every two levels
+    private val delay: Long = 500 // how often does the progressbar update
+    private var currentLevel = 1 //lv the player is on
+    private var subLevel = 0 // for the progressbar and to change level length
+    private val handlerTimer = Handler()
```

Comment:
"'constructor Handler()' is deprecated. Deprecated in Java"
You can fix it by changing it to:
```kt
private val handlerTimer = Handler(Looper.getMainLooper())
```
and ofc. importing Looper